### PR TITLE
FIX DebugBar now trims "extra" section of PHP_VERSION from PHP version information for readability

### PR DESCRIPTION
**Pull request by robbieaverill**
Resolves https://github.com/lekoala/silverstripe-debugbar/issues/85

### CODE REVIEW

File: code/Collector/PhpInfoCollector.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    /**
+     * Given a PHP_VERSION constant value, trim any "extra" meta information from the end, returning the major, minor
+     * and patch release of the version. Will fall back to returning the input if the matching fails.
+     *
+     * @param string $version
+     * @return string
+     */
+    protected function trimVersion($version)
+    {
+        preg_match('/^(\d+\.\d+\.\d+)/', $version, $matches);
```
Comment (ScopeyNZ) [inline code/Collector/PhpInfoCollector.php]: Pretty sure you don't need the capturing group here - but it's so minor I won't block merge 🙂

**Review by ScopeyNZ — APPROVED** [review]

File: code/Collector/PhpInfoCollector.php (hunk 1 above)
Comment (robbieaverill) [reply code/Collector/PhpInfoCollector.php]: TIL

Comment (ScopeyNZ) [reply code/Collector/PhpInfoCollector.php]: Turns out I don't have the authority to merge (or therefore block it I guess).

**Comment by robbieaverill** [comment]
@lekoala would you mind please adding @ScopeyNZ to our team's contributor access when you have a chance? Thanks!

**Pull request merged**